cohort-evaluator-spark/pom.xml

```diff
@@ -68,6 +68,22 @@
 			<groupId>org.opencds.cqf.cql</groupId>
 			<artifactId>engine</artifactId>
 		</dependency>
+		
+		<dependency>
+			<groupId>ca.uhn.hapi.fhir</groupId>
+			<artifactId>hapi-fhir-base</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>ca.uhn.hapi.fhir</groupId>
+			<artifactId>org.hl7.fhir.r4</artifactId>
+		</dependency>
+		
+		<dependency>
+			<groupId>ca.uhn.hapi.fhir</groupId>
+			<artifactId>hapi-fhir-structures-r4</artifactId>
+			<scope>runtime</scope>
+		</dependency>
 
```

Comment:
We should have a team discussion on the use of HAPI FHIR in this location. I've been very specifically avoiding any HAPI dependencies for the Spark work. One one hand, it is consistent with our work in other areas of the project. On the other hand, I think we would like to move away from HAPI and instead use IBM FHIR for all our work. The IBM FHIR model objects are slightly harder to use, but bring a bunch of improvements in terms of data consistency and validation in addition to the eat your own dogfood benefit.